src/crafting.h

```diff
@@ -3,130 +3,15 @@
 #define CRAFTING_H
 
 #include "item.h"         // item
-#include "cursesdef.h"    // WINDOW
-#include "requirements.h"
+#include "recipe.h"
```

Comment:
Yeah, that's my laziness. A forward declaration requires includes in the places where the `recipe` is actually utilized, not merely mentioned. Will do.